tests/create-projects/create-projects_test.go

```diff
@@ -1,31 +1,77 @@
 package create_projects
 
 import (
-	"fmt"
-	"os"
-	"os/exec"
-	"path"
-	"runtime"
-	"testing"
+    "fmt"
+    projectClientV1 "github.com/openshift/client-go/project/clientset/versioned/typed/project/v1"
+    "gotest.tools/assert"
+    meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+    "k8s.io/client-go/tools/clientcmd"
+    "os"
+    "os/exec"
+    "path"
+    "path/filepath"
+    "runtime"
+    "testing"
 )
 
+func homeDir() string {
+    if h := os.Getenv("HOME"); h != "" {
+        return h
+    }
+    return os.Getenv("USERPROFILE") // windows
+}
+
 func TestCreateProjectWithoutProjectId(t *testing.T) {
-	_, filename, _, _ := runtime.Caller(0)
-	dir := path.Join(path.Dir(filename), "..", "..")
-	cmd := exec.Command("sh", fmt.Sprintf("%s/create-projects/create-projects.sh", dir))
-	out, err := cmd.CombinedOutput()
-	if err == nil {
-		t.Errorf("Execution of `create-project.sh` must fail if no PROJECT_ID is set: %s", out)
-	}
+    _, filename, _, _ := runtime.Caller(0)
+    dir := path.Join(path.Dir(filename), "..", "..")
+    cmd := exec.Command("sh", fmt.Sprintf("%s/create-projects/create-projects.sh", dir))
+    out, err := cmd.CombinedOutput()
```

Comment:
I recommend splitting the output to make it easier to investigate issues.

Reply:
Fixed